VideoWeb/VideoWeb.Common/Security/BaseServiceTokenHandler.cs

```diff
@@ -30,12 +30,12 @@ public abstract class BaseServiceTokenHandler : DelegatingHandler
         protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request,
             CancellationToken cancellationToken)
         {
-            var token = GetServiceToServiceToeken();
+            var token = GetServiceToServiceToken();
```

Comment:
this should not have been merged originally. ridiculous!

Reply:
ToeKen. Seriously? GET. A. GRIP.